cclient/serialization/json/get_balances.c

```diff
@@ -56,29 +56,92 @@ retcode_t json_get_balances_serialize_request(
   return ret;
 }
 
-retcode_t json_get_balances_deserialize_response(serializer_t const *const s,
-                                                 char const *const obj,
-                                                 get_balances_res_t *out) {
+retcode_t json_get_balances_deserialize_request(serializer_t const *const s,
+                                                char const *const obj,
+                                                get_balances_req_t *const req) {
   retcode_t ret = RC_OK;
   cJSON *json_obj = cJSON_Parse(obj);
   cJSON *json_item = NULL;
   log_info(json_logger_id, "[%s:%d] %s\n", __func__, __LINE__, obj);
 
-  if (json_obj == NULL) {
-    log_error(json_logger_id, "[%s:%d] %s\n", __func__, __LINE__,
-              STR_CCLIENT_JSON_PARSE);
-    cJSON_Delete(json_obj);
-    return RC_CCLIENT_JSON_PARSE;
+  JSON_CHECK_ERROR(json_obj, json_item, json_logger_id);
+
+  ret = json_get_uint8(json_obj, "threshold", &req->threshold);
```

Comment:
Could you please help to change `threshold` as uint8_t and the comment in `cclient/request/get_balances.h`?
```
typedef struct {
  /**
   * The confirmation threshold between 0 and 100(inclusive).
   *  Should be set to 100 for getting balance by counting only confirmed transactions.
   */
  uint8_t threshold;
```

Reply:
sure